### ISSUE TITLE
replace data-raw getter with bowerbird config

### ISSUE THREAD

**Issue by mdsumner** [mdsumner/orsifronts#4]
- [ ] update to use devtools use_data

**Comment by raymondben** [comment]
I think bowerbird's kind of overkill here ... it only needs to retrieve one file and unzip it. I think what you have is fine!

**Comment by mdsumner** [comment]
yeah  let's just consider this a calibration moment, I will update to use_devtools though :)
